### DISCUSSION TITLE
Gp Pro 11 Black Livestream

### DISCUSSION THREAD

**Discussion by powerswissi** [gopro/labs#471] [Q&A]
Hello there . i did a livestream with the go pro 11 with the qr codes 

It worked very very good but is there a limit in the connection? ive got full coverage of 4g and with the quick app ive got 5000kb. 

With labs the max was 4111kb. Is there any chance to set it to 6000 or more? 

greetings from swiss

**Comment by dnewman-gpsw** [comment]
The bit-rate is dynamic base don the channel, but it has an upper limit. Labs doesn't change this limit, it is using the same mechanism for live streams as the regular FW.  I have added this request to the backlog.

**Comment by powerswissi** [comment]
my Livestream was always in 4111 kb.. on 720p or 1080p .. yesterday with the quick app was 720p with 2500max und 1080 with 5000max

my speedtest goes to 6000 upload with full connection

**Reply by dnewman-gpsw** [reply]
I have confirmed this and determined a new feature for a future release.  Setting a livestream bit-rate should be possible.

**Reply by ghost** [reply]
Wooohooo, finally live-streaming gets improvment - lookin forward too it, thank you so much!

**Reply by dnewman-gpsw** [reply]
Today's release for HERO11 supports setting the max bitrate for livestream with the oMBITL=x command.

**Comment by powerswissi** [comment]
thanks for the 8000 kbits.. how can i set it now? sry i am new on this

**Reply by dnewman-gpsw** [reply]
If you always want to try for 8Mb/s, use the command !MBITL=8 with a QR Code like this:
![image](https://github.com/gopro/labs/assets/10856084/8c28e2c6-2d84-4646-b0ff-328872614da5)

All !M style commands are permanent, only need to set this once, and the camera will remember.  If your livestreams have issues at the bit-rate, use !MBITL=0 (to erase the command) or !RESET (to erase of Labs mods.)

**Comment by powerswissi** [comment]
ah okay.. thank you .. i will try.. Thank you
